frontend/packages/system/src/Notifications/ListView/ListView.tsx

```diff
@@ -13,16 +13,36 @@ import {
   createTableStore,
   useColumns,
 } from '@common/ui';
-import { NotificationsModal } from '../Modals/NotificationsModal';
 import { useEditModal, useQueryParamsState } from '@common/hooks';
 import { NotificationConfigRowFragment, useNotificationConfigs } from '../api';
 import { useDeleteNotificationConfig } from '../api/hooks/useDeleteNotificationConfig';
+import { ConfigKind, useNavigate } from 'packages/common/src';
```

Comment:
importing directly from `packages/` like this is fine in the notify app - but breaks tests in open mSupply 🤷 
It makes this suggestion optional - though the import is neater looking if you use the package name:
```suggestion
import { ConfigKind, useNavigate } from '@notify-frontend/common';
```

Reply:
I rely too much on the auto importer :(
Thanks for picking this up!